src/api/ci/jenkins/utils/jenkins.utils.ts

```
@@ -0,0 +1,264 @@
+
+    return 'UNKNOWN';
+  }
+
+  /**
+   * Check if build has git information
+   */
+  private static hasGitInformation(build: any): boolean {
+    return build.actions && build.actions.some((action: any) =>
+      action._class?.includes('git') || action.lastBuiltRevision || action.buildsByBranchName);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)